src/OpenTelemetry.Instrumentation.Process/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Update OpenTelemetry API to 1.4.0-rc.1 ([#820](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/820))
```

Comment:
nit: I think we usually have been entering the PR link in CHANGELOG in a new line.

Reply:
Sure, I will fix it next time, Now, it was just copy paste from the previous record in the same file.